libs/events/src/types/orders.ts

```diff
@@ -1,15 +1,19 @@
-export interface OnPostedOrderPayload {
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
+
+export interface OrderUidInChain {
   orderUid: string
+  chainId: SupportedChainId
   // TODO: Potentially add all order info here, but lets keep it minimal for now
 }
 
-export interface OnExecutedOrderPayload {
-  orderUid: string
-  // TODO: Potentially add all order info here, but lets keep it minimal for now
+export type OnPostedOrderPayload = OrderUidInChain
+export type OnExecutedOrderPayload = OrderUidInChain
+
+export interface OnPostedEthFlowOrderPayload extends OrderUidInChain {
+  txHash: string
 }
```

Comment:
Nitpick, because I like types, and the other payloads are types already

```suggestion
export type OnPostedEthFlowOrderPayload = OrderUidInChain & {
  txHash: string
}
```

Reply:
hehe, i like more interface for this case, but i will do it your way :)